### PR TITLE
Switching over Docker Hub image

### LINKED ISSUE #574: Switch default container registry to Docker Hub rather than quay.io

### PR DESCRIPTION
**Pull request by prometherion**
Closes #574.

### CODE REVIEW

File: charts/capsule/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 manager:
   image:
-    repository: quay.io/clastix/capsule
+    repository: clastix/capsule
     pullPolicy: IfNotPresent
     tag: ''
```
Comment (ptx96) [inline charts/capsule/values.yaml:10]: We should also change kubectl [image registry](https://github.com/clastix/capsule/pull/575/files#diff-8facec49e536e52cd2044382bdb9bbd0227328f1977cfee6dc55f73886a67b7bR43), don't we ?

**Review by ptx96 — CHANGES_REQUESTED** [review]
Just a note on kubectl container, otherwise, LGTM!

File: charts/capsule/values.yaml (hunk 1 above)
Comment (prometherion) [reply charts/capsule/values.yaml:10]: Nice spot! Created the Docker Hub [repository](https://hub.docker.com/repository/docker/clastix/kubectl), copying in the meanwhile the images from Quay.io to Docker.io!

**Review by ptx96 — APPROVED** [review]
LGTM

**Pull request merged**